languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt

```diff
@@ -20401,4 +20401,5 @@ Bear Stearns
 Horst Görtz
 SteamOS
 WW International
-Jens Rommel
\ No newline at end of file
+Jens Rommel
+David Copperfield
```

Comment:
It's okay. Now it is a duplicate, and it will be removed when we check for duplicates.